src/hooks/useTxNotifications.ts

```diff
@@ -68,6 +76,44 @@ const useTxNotifications = (): void => {
       unsubFns.forEach((unsub) => unsub())
     }
   }, [dispatch, safeAddress, chain?.shortName])
+
+  /**
+   * Show a notification when a transaction is awaiting confirmation on Safe load
+   */
+
+  const { page } = useTxQueue()
+  const pendingTxs = useAppSelector(selectPendingTxs)
+  const notifications = useAppSelector(selectNotifications)
+
+  useEffect(() => {
+    const items = page?.results?.filter(isTransactionListItem)
+
+    if (!items || items.length === 0) {
+      return
+    }
+
+    const { transaction } = items[0]
+
+    if (transaction.txStatus !== TransactionStatus.AWAITING_CONFIRMATIONS || pendingTxs[transaction.id]) {
```

Comment:
Don't we also need to check that it's awaiting _your_ signature?

Reply:
Good point. I'll adjust it now.